src/session/SessionMiddleware.ts

```diff
@@ -28,13 +29,10 @@ function sessionRequestHandler(config: CookieConfig, sessionStore: SessionStore)
 
         if (sessionCookie) {
 
-            console.log("Got a session cookie.");
-            console.log(`REQUEST: ${request.url}`);
-            console.log(`COOKIE: ${sessionCookie}`);
+            loggerInstance().info(`Session cookie - REQUEST: ${request.url} - COOKIE: ${sessionCookie}`);
```

Comment:
I think here you can just do `loggerInstance().infoRequest(request)` and then `loggerInstance().debug('sessionCookie=${sessionCookie}')`

Reply:
```suggestion
           loggerInstance().infoRequest(request);
           loggerInstance().debug(`sessionCookie=${sessionCookie}`);
```

Reply:
What that `infoRequest` method does?  Unfortunately name does not tell much.

Reply:
logs the request call in a standard way with `info` level. Tbf doing `loggerInstance().request(request)` is sufficient

Reply:
By looking at the source code I doubt `loggerInstance().request(request)` is useful. It seems to be another log level coloured in a specific way.

To avoid extracting URL from request manually there is `infoRequest` method that could be used in the following way:

```
infoRequest(request, 'some message')
```

Reply:
.request() takes a message: string as parameter, not a request. 

.infoRequest() takes both a message and a request, not sure how much shorter it is than what we did already

Reply:
My bad then

Reply:
{"created":"2020-04-30T09:15:43.707Z","event":"info","namespace":"node-session-handler","data":{"message":"some message"}}

This is what infoRequest(request, 'some message') returns. What is the point of the request param? Seems to store as some sort of metadata?

Reply:
There's some metadata that the CH support team needs apparently. There's also this middleware you could apply which logs the beginning and end of the request automatically https://github.com/companieshouse/ch-structured-logging-node/blob/master/src/MiddlewareFactory.ts

Reply:
I do not think middleware should be applied at the library level. Is logs request start and end which makes only sense in apps.

Reply:
```
    public infoRequest(request: Request, message: string) {
        this.logger.info(message, getRequestMetaData(request));
    }
```

☝️ Above method should in theory log error message and additional request metadata extracted via `getRequestMetaData` method which really produces JSON object with path and method. I would expect to see these two things in log you printed but it seems to not work.